src/style/mixins.scss

```diff
@@ -5,3 +5,20 @@
     }
   }
 }
+
+
+@mixin is-elevated-clickable() {
```

Comment:
This is a pretty good example of a `refactor` type commit. If it would have replaced existing copy-and-pasted code in several places with this mixin, it would have been a spot-on `refactor`. Since it only adds a mixin for use in upcoming commits, this could also be marked as a `chore`, it doesn't really matter.